pkg/api/vlabs/const.go

```diff
@@ -67,3 +71,8 @@ const (
 	// ManagedDisks means that the nodes use managed disks for their os and attached volumes
 	ManagedDisks = "ManagedDisks"
 )
+
+// Network policy
+var (
+	NetworkPolicyValues = [...]string{"", "none", "azure", "calico"}
```

Comment:
Huh. I didn't even know that was valid syntax... `[]string{"literal", "literal"}` is more idiomatic Go.